src/underline.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * @module basic-styles/underline
+ */
+
+import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
+import UnderlineEngine from './underlineengine';
+import ButtonView from '@ckeditor/ckeditor5-ui/src/button/buttonview';
+import underlineIcon from '../theme/icons/underline.svg';
+
+/**
+ * The underline feature. It introduces the Underline button and the <kbd>Ctrl+U</kbd> keystroke.
+ *
+ * It uses the {@link module:basic-styles/underlineengine~UnderlineEngine underline engine feature}.
+ *
+ * @extends module:core/plugin~Plugin
+ */
+export default class Underline extends Plugin {
+	/**
+	 * @inheritDoc
+	 */
+	static get requires() {
+		return [ UnderlineEngine ];
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	static get pluginName() {
+		return 'Underline';
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	init() {
+		const editor = this.editor;
+		const t = editor.t;
+		const command = editor.commands.get( 'underline' );
+		const keystroke = 'CTRL+U';
```

Comment:
Yes, there's shortcut for it in CKE4 – `Ctrl+U`. The same shortcut is also available in Word, so it's probably safe to  suppose that it's a _de facto_ standard.